### PR TITLE
Fix #137

### LINKED ISSUE #137: Check documentation for new battery feature
The new battery featured added in #135

* API reference - tidy Battery class docs if needed and add Vehicle.battery member
* Guide/Examples - make sure that this feature is documented in the guide and examples where relevant - e.g. http://python.dronekit.io/guide/vehicle_state_and_parameters.html

### PR DESCRIPTION
**Pull request by maruina**
Change "-1" with "None".
Fixed the print issue.

```
print " Battery: %s" % v.battery
Battery: Battery voltage: 11.997, current: None, level: None
```

### CODE REVIEW

**Comment by mrpollo** [comment]
thanks @maruina, @hamishwillee code looks fine, can you make sure problems discussed on #137 are fixed?

**Comment by hamishwillee** [comment]
I ran some tests and it does work as promised and addresses #137. The docs need some work as discussed, but I can create a patch for that as soon as you've merged.

**Comment by mrpollo** [comment]
hey @hamishwillee can you share your test script with us, I'm going to have @tcr3dr help check this

**Comment by hamishwillee** [comment]
All I did was add the following line to the (vehicle_state)[http://python.dronekit.io/examples/vehicle_state.html#source-code] source code near the other parameters:

```
print " Battery: %s" % v.battery
```
Then I ran the script at various states in the running code.

Of course I first had to set up a local test build, which I did as per our google doc on this topic and copy in the changes in the file for __init__.py (might have been easier to sync to the official branch)

File: droneapi/lib/__init__.py [hunk 1]
```diff
@@ -173,17 +173,20 @@ class Battery(object):
     """
     Battery info available
 
-    :param battery_voltage: Battery voltage in millivolts
-    :param battery_current: Battery current, in 10 * milliamperes, -1: autopilot does not measure the current
-    :param battery_level: Remaining battery energy, -1: autopilot estimate the remaining battery
+    :param voltage: Battery voltage in millivolts
+    :param current: Battery current, in 10 * milliamperes, -1: autopilot does not measure the current
+    :param level: Remaining battery energy, -1: autopilot estimate the remaining battery
```
Comment (hamishwillee) [inline droneapi/lib/__init__.py]: The changes I plan to make to this section are:

- add full stops :-)
- Now these return ```None``` document that

So:
```
:param voltage: Battery voltage in millivolts.
:param current: Battery current, in 10 * milliamperes. ``None`` if the autopilot does not support current measurement.
:param level: Remaining battery energy. ``None`` if the autopilot cannot estimate the remaining battery.
```

Note the change to description of "level". This is an " educated guess" on the behaviour - the original text (from the specification) does not make any sense.

Comment (maruina) [inline droneapi/lib/__init__.py]: I fixed it.
Hope everything is fine! :)

Comment (hamishwillee) [inline droneapi/lib/__init__.py]: Awesome. Thank you. @mrpollo - ready to integrate.

**Comment by mrpollo** [comment]
:+1: thanks guys this is great

**Pull request merged**